modules.d/35network-connman/module-setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+    require_binaries sed grep || return 1
+
+    # do not add this module by default
+    return 255
+}
+
+# called by dracut
+depends() {
+    echo dbus systemd bash
+    return 0
+}
+
+# called by dracut
+installkernel() {
+    return 0
+}
+
+# called by dracut
+install() {
```

Comment:
Here you are missing optional install of /usr/lib/connman/wired-setup in conjuction with the ExecStartPre=-/usr/lib/connman/wired-setup in cm-initrd.service. 

If it's not needed remove that ExecStartPre line from the type unit.

Reply:
> If it's not needed remove that ExecStartPre line from the type unit.

Removed the ExecStartPre line.